modular_skyrat/modules/modular_items/lewd_items/code/technical_stuff_for_lewd.dm

```diff
@@ -50,11 +50,7 @@
 
 /obj/item/storage/box/milking_kit/PopulateContents()
 	var/static/items_inside = list(
-		/obj/item/milking_machine/constructionkit = 1,
-		/obj/item/reagent_containers/glass/beaker = 1,
```

Comment:
tbh i think the rest is fine, just the tools are bad imo

Reply:
beakers being available in permabrig means clever prison botanists can splash tons of shit onto guards if they know what they're doing, and an upgraded power cell means prisoners can make cattle prods

Reply:
make it the shitty ass battery and a small beaker?

Reply:
also, prisons with grinders start with a large beaker

I'm not too bothered personally